modules/redisearch.ts

```diff
@@ -540,289 +562,441 @@ export class Redisearch extends Module {
 
 /**
  * The 'FT.CREATE' additional optional parameters
- * @param filter The expression of the 'FILTER' parameter. is a filter expression with the full RediSearch aggregation expression language.
- * @param payloadField The field of the 'PAYLOAD' parameter. If set indicates the document field that should be used as a binary safe payload string to the document, that can be evaluated at query time by a custom scoring function, or retrieved to the client.
- * @param maxTextFields The 'MAXTEXTFIELDS' parameter. For efficiency, RediSearch encodes indexes differently if they are created with less than 32 text fields.
- * @param noOffsets The 'NOFFSETS' parameter. If set, we do not store term offsets for documents (saves memory, does not allow exact searches or highlighting).
- * @param temporary The 'TEMPORARY' parameter. Create a lightweight temporary index which will expire after the specified period of inactivity.
- * @param nohl The 'NOHL' parameter. Conserves storage space and memory by disabling highlighting support. If set, we do not store corresponding byte offsets for term positions.
- * @param noFields The 'NOFIELDS' parameter. If set, we do not store field bits for each term.
- * @param noFreqs The 'NOFREQS' parameter.  If set, we avoid saving the term frequencies in the index.
- * @param skipInitialScan The 'SKIPINITIALSCAN' parameter. If set, we do not scan and index. 
- * @param prefix The 'PREFIX' parameter. tells the index which keys it should index.
- * @param prefix.count The count argument of the 'PREFIX' parameter. 
- * @param prefix.name The name argument of the 'PREFIX' parameter. 
- * @param language The 'LANGUAGE' parameter.  If set indicates the default language for documents in the index.
- * @param languageField The 'LANGUAGE_FIELD' parameter. If set indicates the document field that should be used as the document language.
- * @param score The 'SCORE' parameter. If set indicates the default score for documents in the index.
- * @param scoreField The 'SCORE_FIELD' parameter. If set indicates the document field that should be used as the document's rank based on the user's ranking. 
- * @param stopwords The 'STOPWORDS' parameter. If set, we set the index with a custom stopword list, to be ignored during indexing and search time.
- * @param stopwords.num The num argument of the 'STOPWORDS' parameter. 
- * @param stopwords.stopword The stopword argument of the 'STOPWORDS' parameter.
- */
-export type FTCreateParameters = {
+*/
+export interface FTCreateParameters {
+    /**
+    * The expression of the 'FILTER' parameter. is a filter expression with the full RediSearch aggregation expression language.
+    */
```

Comment:
Why is this parameter defining better? I was perfectly fine as before.. Unless it's better, please keep it as was.

Reply:
By default, vsc reads docs using the jsdoc specification. And properties should be documented this way in order to be shown correctly.
![1](https://user-images.githubusercontent.com/40274710/131303723-32417ffe-06a0-4016-b1d2-e858000a74cd.png)